modules/account-lib/src/coin/algo/ifaces.ts

```diff
@@ -0,0 +1,46 @@
+import algosdk from 'algosdk';
```

Comment:
in some coins we call this file `iface.ts`, we should pick one and make it standard